core/src/gql/franchise/franchise.object.ts

```diff
@@ -2,15 +2,15 @@ import { Field, ObjectType } from '@nestjs/graphql';
 
 @ObjectType('Franchise')
 export class FranchiseObject {
-  @Field()
-  id: string;
+	@Field()
```

Comment:
Not necessary to address in this PR, but leaving it as a reminder: we should have a standard formatter for this project and it should be run as a githook. I should not be having to review whitespace, as much as possible (all of us shouldn't, not just me!) 
<img width="825" alt="image" src="https://github.com/user-attachments/assets/81767751-14e6-4105-8d23-75470983d169" />